### ISSUE TITLE
Add amnesty international CH to "they use plone"

### ISSUE THREAD

**Issue by tisto** [plone/plone.de#64]
https://www.amnesty.ch/en

**Comment by jonaspiterek** [comment]
@tisto done. also fixed that some images where not shown with #68

**Comment by TimoBroeskamp** [comment]
This is done and can be closed

**Comment by stevepiercy** [comment]
It would be good to add:

https://www.amnesty.ch/en

to:

- https://github.com/collective/awesome-volto/
- https://plone.org/why-plone/they-use-plone

Do we have more information about the site, such as who created it?

@plone/marketing-communications

**Comment by tisto** [comment]
@stevepiercy I agree. This is a good success story. Back in the day, 4teamwork did this. These days maybe webcloud7?

**Comment by stevepiercy** [comment]
@tisto see https://github.com/plone/plone.org/issues/238, where the planning is taking place. I'm not leading the task, but merely connecting the dots. I realized that both of these sites as well as the Volto READMEs weren't being kept in sync with https://github.com/collective/awesome-volto/. plone.org should be authoritative, and other sites should sync with it. This would be a nice task to complete at Bucharest.
